src/StyleableBannerWithLink/index.tsx

```diff
@@ -105,6 +105,8 @@ const StyleableBannerWithLink: React.FC<Props> = (props: Props) => {
 
                         {highlight ? (
                             <>
+                                <br />
```

Comment:
I wonder if it would be better to use a container with margin to manage the spacing here? Does this give the right spacing at all screen widths? (Just thinking - if not, I think it's easier to adapt to different screen widths with a marging vs a `br` tag).